docs/Help.md

```diff
@@ -11,6 +11,7 @@ changed.  Also a good place to look for new goodies.
 ## Help topics
 
 * [Travis CI file explained](TravisFileExplained.md): every line of the `.travis.yml` file explained.
```

Comment:
Not belonging strictly to this issue.. but the next line introduced (for gha) called my attention over it... shouldn't be `.travis.dist.yml` here (to be well paired with the gha one)

Reply:
Well spotted, you are right. Changed.